dlrover/python/master/node/dist_job_manager.py

```diff
@@ -28,8 +28,12 @@
     NodeType,
 )
 from dlrover.python.common.global_context import Context
+from dlrover.python.common.grpc import ParallelConfig
 from dlrover.python.common.log import default_logger as logger
 from dlrover.python.common.node import Node, NodeGroupResource
+from dlrover.python.master.hyperparams.local_strategy_generator import (
```

Comment:
Do you miss the `local_strategy_generator`?
It is better to rename `local_strategy_generator` to `strategy_generator` because the local job or distributed job
can both use it.